### PR TITLE
Update russian translation

### PR DESCRIPTION
**Pull request by IndeIrae**
It's that time of the year again.
- Anti-sub -> ППЛ (противоподлодочн[ый/ая/ое]);
- Undersea Fire -> подводная атака;
- Other up-to-date catching up stuff.

### CODE REVIEW

File: LuaUI/Configs/lang/units.ru.json [hunk 1]
```diff
@@ -926,37 +926,37 @@
...
 	"shiptorpraider" : {
 		"name" : "Охотник",
-		"description" : "Торпедный фрегат (рейдер)",
+		"description" : "Торпедный рейдер (ППЛ)",
 		"helptext" : "Охотник (Hunter) - это мобильный противолодочный юнит. Его оружие эффективно поражает над- и подводные цели."
 	},
 	"shipskirm" : {
 		"name" : "Мистраль",
-		"description" : "Ракетный крейсер (стрелок/артиллерия)",
+		"description" : "Ракетный корабль",
```
Comment (sprunk) [inline LuaUI/Configs/lang/units.ru.json:939]: lel

Comment (IndeIrae) [reply LuaUI/Configs/lang/units.ru.json:939]: closer to how it is since ship desc changes :P

File: LuaUI/Configs/lang/units.ru.json [hunk 2]
```diff
@@ -357,7 +357,7 @@
 	"dronefighter" : {
 		"name" : "Щупальце",
 		"description" : "Боевой дрон",
-		"helptext" : "Щупальце (Spicula) - это многоцелевой боевой дрон, силовые лучи которого эффективны как против наземных, так и воздушных целей."
+		"helptext" : "Щупальце (Spicula) - это многоцелевой боевой дрон, силовые лучи которого эффективны как против наземных, так и против воздушных целей."
```
Comment (sprunk) [inline LuaUI/Configs/lang/units.ru.json:360]: Already says против a few words earlier, is this repetition natural?

File: LuaUI/Configs/lang/units.ru.json [hunk 3]
```diff
@@ -817,7 +817,7 @@
 	"cloakaa" : {
 		"name" : "Гремлин",
 		"description" : "Робот ПВО",
-		"helptext" : "Быстрый и довольной выносливый для своей цены, Гремлин (Gremlin) - отличный выбор мобильного средства противовоздушной обороны. Он невидим, что позволяет ему обеспечивать неожиданную противовоздушную защиту и избегать стычки с наземными войсками противника, против которых он беззащитен."
+		"helptext" : "Быстрый и и довольно выносливый для своей цены, Гремлин (Gremlin) - отличный выбор мобильного средства противовоздушной обороны. Он невидим, что позволяет ему обеспечивать неожиданную противовоздушную защиту и избегать стычки с наземными войсками противника, против которых он беззащитен."
```
Comment (sprunk) [inline LuaUI/Configs/lang/units.ru.json:820]: и и

File: LuaUI/Configs/lang/units.ru.json (hunk 1 above)
Comment (Anarchid) [reply LuaUI/Configs/lang/units.ru.json:939]: approve

File: LuaUI/Configs/lang/units.ru.json (hunk 2 above)
Comment (Anarchid) [reply LuaUI/Configs/lang/units.ru.json:360]: It's natural yeah

**Pull request merged**